controllers/CommentController.js

```diff
@@ -93,7 +93,7 @@ class CommentController {
         isArchived: true,
```

Comment:
I think line 92 should have an `await` in front of `comments.update`

Reply:
thanks